sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts

```diff
@@ -248,6 +263,26 @@ describe(
       expect(bdReceiptAwards).toStrictEqual({
         BCSG: 599,
       });
+      // Notification record.
+      const notification = await db.notification.findOne({
+        select: {
+          messagePayload: true,
+          createdAt: true,
+        },
+        where: {
+          createdAt: MoreThan(currentTestTime),
+        },
+      });
+      const emailContact = notification.messagePayload["email_address"];
```

Comment:
Can the assertion be changed as in the example below?

https://github.com/bcgov/SIMS/blob/16fd41efd70b58dcde570a3edab89f95221b4b28/sources/packages/backend/apps/api/src/route-controllers/restriction/_tests_/e2e/restriction.aest.controller.addStudentProvincialRestriction.e2e-spec.ts#L59-L67